contracts/AMMWrapper.sol

```diff
@@ -231,7 +214,11 @@ contract AMMWrapper is IAMMWrapper, StrategyBase, ReentrancyGuard, BaseLibEIP712
      * @dev internal function of `trade`.
      * It executes the swap on chosen AMM.
```

Comment:
This line of comment seems outdated. Could be removed.

Reply:
OK!

Reply:
I changed it to the following statement:
```
    /**
     * @dev internal function of `trade`.
     * It verifies the spender permission and transfer assets from the user to this contract.
     */
```